cmd/contour/cli.go

```diff
@@ -45,21 +44,20 @@ func (c *Client) dial() *grpc.ClientConn {
 	case c.CAFile != "" || c.ClientCert != "" || c.ClientKey != "":
 		// If one of the three TLS commands is not empty, they all must be not empty
 		if !(c.CAFile != "" && c.ClientCert != "" && c.ClientKey != "") {
-			log.Fatal("You must supply all three TLS parameters - --cafile, --cert-file, --key-file, or none of them.")
+			kingpin.Fatalf("You must supply all three TLS parameters - --cafile, --cert-file, --key-file, or none of them.")
 		}
 		// Load the client certificates from disk
 		certificate, err := tls.LoadX509KeyPair(c.ClientCert, c.ClientKey)
-		check(err)
-
+		kingpin.FatalIfError(err, "")
```

Comment:
Please check https://play.golang.org/p/1WiDkfYYoLp for the output.

Reply:
This should be 
```
kingpin.FatalIfError(err, "failed to load certificate")
```

Reply:
Made the changes to all FtalIfError. Added Error messages